server/prisma/datamodel.prisma

```diff
@@ -0,0 +1,67 @@
+enum TaskStatus {
+  COMPLETED
+  UNCOMPLETED
+}
+
+enum UserRole {
+  NEWBIE
+  BUDDY
```

Comment:
Ok, agree. Sounds good for the future :)

Reply:
OK, I agree on keep things easy for now and implement more down the line consistently with new features introduction.
Regard the Confluence page, I would like to have this doc aligned with the SRD. If not inside, at least linked to it, so that amendments of the requirements are tracked all in one place. Child pages will create mess IMHO, so maybe better to use clear versioning like the SRD. I will speak with @michaelmoney about that when he'll be back.

I'm opening a Jira task just to track the new requirement for the future: https://digitalrig.atlassian.net/jira/software/projects/BUD/boards/497/backlog?selectedIssue=BUD-115